src-vue/src/views/DeveloperView.vue

```diff
@@ -4,6 +4,39 @@
             <el-alert title="Warning" type="warning" :closable="false" show-icon>
                 This page is designed for developers. Some of the buttons here can break your Northstar install if you do not know what you're doing!
             </el-alert>
+
+            <el-button type="primary" @click="getTags">
+                Get tags
+            </el-button>
+
+            <el-select v-model="firstTag" class="m-2" placeholder="First tag">
+                <el-option
+                    v-for="item in ns_release_tags"
+                    :key="item.name"
+                    :label="item.name"
+                    :value="item.name"
```

Comment:
I cannot get it to sort list by `item.name` but on click have `item` be the selected object (as opposed to `item.name`).

Any help on that would be appreciated <3

Reply:
Why do you want to do this? Do you want to put some stuff inside your `item` instances?

Reply:
The item is of type [tag](https://github.com/R2NorthstarTools/FlightCore/pull/188/files#diff-adfc12dc30d4e3e06e9a49d216f3db3ca6cf3b493dc344db7793fb814a48d8ebR3) which right now only has a single member called `name`. For the future, I might also consider adding information to the type concerning repo or commit hash.

Naturally this doesn't need to be shown in the dropdown menu but I still need to pass the full object to [`compareTags`](https://github.com/R2NorthstarTools/FlightCore/pull/188/files#diff-0944e810871d55931e96ce560916f7fe1048c4a18cfbd3ab18a8a83dd59c968aR300-R317) which in turn passes it to the backend function.

And I cannot figure out how to get the list sorted by `tag.name` but on selecting an element of the list set the value of [`firstTag`](https://github.com/R2NorthstarTools/FlightCore/pull/188/files#diff-0944e810871d55931e96ce560916f7fe1048c4a18cfbd3ab18a8a83dd59c968aR127-R134) and [`secondTag`](https://github.com/R2NorthstarTools/FlightCore/pull/188/files#diff-0944e810871d55931e96ce560916f7fe1048c4a18cfbd3ab18a8a83dd59c968aR135-R141) to the full `tag` object, not just the content of the `name` field... ^^"

Reply:
[According to the documentation](https://element-plus.org/en-US/component/select.html#basic-usage), selector values must be objects having two keys only, `label` and `value`.
Maybe you can put `ns_release_tags` objects as `value`s?

Reply:
@GeckoEidechse did you manage to solve this issue?

Reply:
I attempted, saw that it can work but needs some refactoring, and then got busy with something else ^^"

Reply:
Should now be fixed with 5f636608ee44bce09e7778115ac07877fb5d978a and 313585e952f3245cda7a2a8c97bf7dcc29a7cf2a ^^